src/inject/dynamic-theme/index.ts

```diff
@@ -108,11 +109,22 @@ function createStaticStyleOverrides() {
 
     const variableStyle = createOrUpdateStyle('darkreader--variables');
     const selectionColors = getSelectionColor(filter);
-    const {darkSchemeBackgroundColor, darkSchemeTextColor, lightSchemeBackgroundColor, lightSchemeTextColor} = filter;
+    const {darkSchemeBackgroundColor, darkSchemeTextColor, lightSchemeBackgroundColor, lightSchemeTextColor, mode, brightness, contrast, sepia, grayscale} = filter;
+    let schemeBackgroundColor = mode === 0 ? lightSchemeBackgroundColor : darkSchemeBackgroundColor;
+    let schemeTextColor = mode === 0 ? lightSchemeTextColor : darkSchemeTextColor;
+    if (brightness !== 100 || contrast !== 100 || sepia !== 0 || grayscale !== 0) {
```

Comment:
Worth replacing this block with
```javascript
schemeBackgroundColor = modifyBackgroundColor(parse(schemeBackgroundColor), filter);
schemeTextColor = modifyForegroundColor(parse(schemeTextColor), filter);
```

It looks like shorter and equivalent. You can also remove `...brightness !== 100 || ...` because it looks like micro optimization here (called just 1 time).

Reply:
Forgot that we don't need to check because modifying the pole color with default settings will return the pole color.